client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/MlClientDocumentationIT.java

```diff
@@ -3466,28 +3466,28 @@ public void testEvaluateDataFrame_Classification() throws Exception {
                         .endObject()
                     .endObject()
                     .endObject());
-        TriFunction<String, String, Double, IndexRequest> indexRequest = (actualClass, predictedClass, p) -> {
+        TriFunction<String, String, String, IndexRequest> indexRequest = (actualClass, predictedClass, otherClass) -> {
             return new IndexRequest()
                 .source(XContentType.JSON,
                     "actual_class", actualClass,
                     "predicted_class", predictedClass,
                     "ml.top_classes", List.of(
-                        Map.of("class_name", predictedClass, "class_probability", p),
-                        Map.of("class_name", "other", "class_probability", 1 - p)));
+                        Map.of("class_name", predictedClass, "class_probability", 0.9),
+                        Map.of("class_name", otherClass, "class_probability", 0.1)));
```

Comment:
Honestly, I think it would be good to have more than two classes here to really represent the multi-class case.

Reply:
Done.